### PR TITLE
Make dependabot ignore Quarkus 2

### PR DESCRIPTION
**Pull request by mpeddada1**
Quarkus 2.x version update PRs are still being created by dependabot. Follow up to #294.

### CODE REVIEW

**Review by suztomo — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ updates:
   - match:
       dependency_name: "io.quarkus:quarkus-universe-bom"
       # Quarkus 2 requires Java 11, which is too high.
-      version_requirement: ">=2.0.0.Final"
+      version_requirement: "2.x"
```
Comment (suztomo) [inline .github/dependabot.yml:12]: it's worth trying.

https://dependabot.com/docs/config-file/#ignored_updates says

> The range format is specific to the package manager

So, we can try `[2.0.0.Final,)` as per [this issue](https://github.com/dependabot/dependabot-core/issues/1845) if "2.x" does not work.

**Review by dzou — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (mpeddada1) [reply .github/dependabot.yml:12]: Nice find! As you mentioned, I will try "2.x" first and go with `[2.0.0.Final,)` if that doesn't work.

**Comment by suztomo** [comment]
It seems working!

Another repository received dependabot PR for the 2.0.2.Final. This repository doesn't receive it. Good.
https://github.com/GoogleCloudPlatform/google-cloud-graalvm-support/pull/161

**Comment by dzou** [comment]
Nice!

**Comment by mpeddada1** [comment]
yay! Thanks for validating it.

**Comment by elefeint** [comment]
:( #299. Let's see if @zhumin8 's comment puts it away for good.

**Pull request merged**